R/utils.R

```diff
@@ -137,6 +137,20 @@ gg_list <- function(...) {
   )
 }
 
+#' List of `rlistings` object
+#'
+#' @param ... (`rlistings`) objects.
+#' @returns a `rl_list` object.
+#' @export
+rl_list <- function(...) {
```

Comment:
maybe, as.rl_list with argument not ... but obj should be easier to use; this can avoid the do.call